imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml

```diff
@@ -2,6 +2,7 @@ post:
   tags:
     - authentication
   summary: Auth Token Exchange
```

Comment:
should it be called `Exchange Auth Token`? we're putting verbs first in all other endpoints names and this one stands out

Reply:
yes